package.json

```diff
@@ -694,13 +694,21 @@
       "default": "https://api.openai.com/v1/chat/completions"
     },
     {
-      "title": "DeepL Auth Key",
-      "name": "deepLAuthKey",
+      "title": "DeepL Api Key",
+      "name": "deepLApiKey",
```

Comment:
Do not modify the name value, which is used to locally store the user's auth key, as it will cause the previous value to be lost.